docs/contributing.md

```diff
@@ -87,54 +162,18 @@ If it shows the `pip` binary at `env/bin/pip` then it worked. 🎉
 
     This makes sure that if you use a terminal program installed by that package (like `flit`), you use the one from your local environment and not any other that could be installed globally.
 
-### Flit
+### Install Typer
 
-**Typer** uses <a href="https://flit.readthedocs.io/en/latest/index.html" class="external-link" target="_blank">Flit</a> to build, package and publish the project.
-
-After activating the environment as described above, install `flit`:
+With the virtual environment activated, you can install Typer in "editable" mode:
 
 <div class="termy">
 
 ```console
-$ pip install flit
-
----> 100%
+$ pip install --editable .
```

Comment:
This does require at least [pip 21.3](https://pip.pypa.io/en/stable/news/#v21-3), as it'll otherwise fail with a note about not finding `setup.py` or `setup.cfg`. So it might be good to add that as a note here as well.